ehitchhiking-ui/src/app/shared/services/request.cache.service.ts

```diff
@@ -0,0 +1,39 @@
+import {HttpRequest, HttpResponse} from '@angular/common/http';
+import {RequestCacheInterface} from '../interfaces/request.cache.interface';
+import {RequestCacheItem} from '../models/request.cache.item';
+
+export class RequestCache implements RequestCacheInterface {
+	static readonly maxCacheAge = 30000;
+	cache: Map<string, RequestCacheItem>;
+
+	constructor() {
+		this.cache = new Map();
+	}
+
+	get(request: HttpRequest<any>): HttpResponse<any> | undefined {
```

Comment:
It is a bad practice to return undefined

Reply:
done